static/js/vue-cdr-access/src/components/filterTags.vue

```diff
@@ -82,52 +82,55 @@ export default {
             return params;
         },
 
-        formatSearchValue(field, type) {
-            if (field === undefined || TYPES[type] === undefined) {
+        formatSearchValue(field, fieldValue) {
```

Comment:
I think you renamed the wrong variable? It looks like `field` is the value of the field, while `type` (now `fieldValue`) is the name of the parameter from the query string.